<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VICON</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botonConectar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>102</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectDisp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="imagen">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="activarDeteccion">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar detección de rostros</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFPS">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="guardarImagen">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar imagen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guardarVideo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar vídeo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="activarColor">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="activarReconocimiento">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar reconocimiento facial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPersona">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Añadir persona:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombrePersona">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
